src/Configuration.php

```diff
@@ -62,6 +62,12 @@ public function instantiator(): callable
 
     public function defaultProxyAutoRefresh(): bool
     {
+        if (null === $this->defaultProxyAutoRefresh) {
+            trigger_deprecation('zenstruck\foundry', '1.9', 'Not explicitly configuring default proxy auto-refresh ("zenstruck_foundry.auto_refresh_proxies" in config or TestState::enableDefaultProxyAutoRefresh()/disableDefaultProxyAutoRefresh()) is deprecated. It will default to true in 2.0.');
```

Comment:
How's this deprecation message?

Reply:
What about:

Not explicitly configuring default proxy auto-refresh is deprecated and will default to true in 2.0. Use "zenstruck_foundry.auto_refresh_proxies" in the bundle config or TestState::enableDefaultProxyAutoRefresh()/disableDefaultProxyAutoRefresh().